frontend/discourse/admin/components/site-settings/secret-list.gjs

```diff
@@ -1,14 +1,13 @@
-/* eslint-disable ember/no-classic-components */
-import Component from "@ember/component";
```

Comment:
I'm guessing this is too niche of a component to be styled by themes/plugins so the dropping the wrapper element of ember component won't break anything?

Reply:
Yep. Couldn't find any reference in the `all-the-*`